### PR TITLE
WIP: Spike/nightwatch setup

### PR DESCRIPTION
**Pull request by bhaskarmelkani**

### CODE REVIEW

File: nightwatch/index.js [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+
+const defaultConfig = Object.assign(baseConfig, paths, {});
+
+module.exports = (userConfig = {}) => {
+
+    /**
+     * Delete these properties so that user cannot overwrite them.
+     * */
+    delete userConfig.src_folders;
+    delete userConfig.page_objects_path;
```
Comment (rbarilani) [inline nightwatch/index.js]: Why?

File: nightwatch/pages/anchor.js [hunk 2]
```diff
@@ -0,0 +1,21 @@
+const util = require('util');
+
+module.exports = {
+  url: function() {
+    return this.api.launchUrl + `/get-started.html#${this.api.globals.pages.anchor.anchorId}`;
```
Comment (rbarilani) [inline nightwatch/pages/anchor.js]: I still see '/get-started.html' here.... I am missing something?

Comment (bhaskarmelkani) [reply nightwatch/pages/anchor.js]: My bad, removed them.

File: nightwatch/index.js (hunk 1 above)
Comment (bhaskarmelkani) [reply nightwatch/index.js]: So that user cannot modify these values from its config file.

**Pull request closed without merging**